Majasdarbs/03-majasdarbs/portfolio/css/normalize.css

```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap');
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family:  'Montserrat';
```

Comment:
adding font fallback (sans-serif for example) is a good idea, in case if fonts doesn't load for some reason.